src/plugins/mf_priority.cpp

```diff
@@ -1046,7 +1104,8 @@ extern "C" int flux_plugin_init (flux_plugin_t *p)
     if (flux_plugin_register (p, "mf_priority", tab) < 0
         || flux_jobtap_service_register (p, "rec_update", rec_update_cb, p)
         || flux_jobtap_service_register (p, "reprioritize", reprior_cb, p)
-        || flux_jobtap_service_register (p, "rec_q_update", rec_q_cb, p) < 0)
+        || flux_jobtap_service_register (p, "rec_q_update", rec_q_cb, p)
+        || flux_jobtap_service_register (p, "rec_p_update", rec_p_cb, p) < 0)
```

Comment:
Yeah absolutely, sorry for the confusion! I was trying to stay under the 80 character limit here but I'll it can be hard for me to look at too, so I should just break it up 👍 will fix this

Reply:
Ah! I can sympathize with that! However, no problem breaking the function across multiple lines!